packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -43,18 +43,28 @@ library LibProving {
         AddressResolver resolver,
         uint256 blockId,
         TaikoData.BlockEvidence memory evidence
-    ) internal {
+    )
+        internal
+    {
         if (
-            evidence.parentHash == 0 || evidence.blockHash == 0
-                || evidence.blockHash == evidence.parentHash || evidence.signalRoot == 0
-                || evidence.gasUsed == 0
+            evidence.parentHash == 0
+            //
```

Comment:
is this `//` added automatically by formatter? hmm not sure if this looks better?

Reply:
I just want to make sure each condition is put in its own line, this is the only way I can make it happen.